src/serac/physics/utilities/functional/domain_integral.hpp

```diff
@@ -197,7 +201,48 @@ void gradient_kernel(const mfem::Vector& dU, mfem::Vector& dR, derivatives_type*
   }
 }
 
-}  // namespace domain_integral
+template < typename T1, typename T2 >
+struct linear_approximation {
+  T1 value;
+  T2 derivative;
+};
+
+template < typename element_type, int dim >
+auto evaluate_shape_functions(tensor < double, dim > xi, tensor< double, dim, dim > J) {
+  if constexpr (element_type::family == Family::HCURL) {
```

Comment:
Have you run style over this?

Reply:
Apparently, my `style` target was not being enabled since we introduced the SERAC_ENABLE_CODE_CHECKS option (prior to that, ENABLE_CLANGFORMAT=ON was sufficient to enable the style checking). After explicitly specifying `SERAC_ENABLE_CODE_CHECKS=ON`, it's working again (thanks Josh).